build.gradle

```diff
@@ -13,6 +13,9 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
+        maven {
+            url "https://maven.google.com"
```

Comment:
required only for early access to v26 and future versions when they are in beta, but 25.3.1 does not require this